cnd/src/storage.rs

```diff
@@ -153,23 +153,23 @@ impl Load<respawn::Swap<halight::Params, herc20::Params>> for Storage {
             Role::Bob => secret_hash.ok_or_else(|| NoSecretHash(id))?.secret_hash.0,
         };
 
-        let swap = respawn::Swap {
-            id,
+        let swap = start_swap::Swap {
             role,
             alpha: build_halight_params(halight, secret_hash, id)?,
             beta: build_herc20_params(herc20, secret_hash, id)?,
+            start_of_swap: swap.start_of_swap,
         };
 
         Ok(swap)
     }
 }
 
 #[async_trait::async_trait]
-impl Load<respawn::Swap<herc20::Params, hbit::Params>> for Storage {
+impl Load<start_swap::Swap<herc20::Params, hbit::Params>> for Storage {
```

Comment:
nit: Feel free not to do this if you want to merge today but this should be pub used in network module then the usage can be `network::Swap` (same for `network::DecisionSwap`).

Reply:
Could you take care of that one @tcharding ? Would appreciate :)